### PR TITLE
Add support to Workflows

### PR DESCRIPTION
**Pull request by AleGladchteinParser**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <Project>
     <PropertyGroup>
-        <Version>3.2.1</Version>
+        <Version>3.1.0</Version>
```
Comment (Armando-Ibarra) [inline Directory.Build.props:3]: I think you might want to rebase your changes with `master`

File: src/CheckoutSdk/Workflows/Four/CreateWorkflowRequest.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using Checkout.Workflows.Four.Conditions.Request;
+using System.Collections.Generic;
+
+namespace Checkout.Workflows.Four
+{
+    public class CreateWorkflowRequest
+    {
+        public string Name { get; set; }
+
+        public IList<WorkflowConditionRequest> Conditions { get; set; } = new List<WorkflowConditionRequest>();
```
Comment (Armando-Ibarra) [inline src/CheckoutSdk/Workflows/Four/CreateWorkflowRequest.cs:11]: Why does this needs to be instantiated by default?

File: src/CheckoutSdk/Workflows/Four/Events/Event.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+
+namespace Checkout.Workflows.Four.Events
+{
+    public class Event
+    {
+        public string Id { get; set; }
+
+        [JsonProperty(PropertyName = "display_name")] public string DisplayName { get; set; }
```
Comment (Armando-Ibarra) [inline src/CheckoutSdk/Workflows/Four/Events/Event.cs:9]: I think this Json property is not needed.

File: src/CheckoutSdk/Workflows/Four/Events/EventTypesResponse.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Checkout.Workflows.Four.Events
+{
+    public class EventTypesResponse
+    {
+        public string Id { get; set; }
+
+        [JsonProperty(PropertyName = "display_name")] public string DisplayName { get; set; }
```
Comment (Armando-Ibarra) [inline src/CheckoutSdk/Workflows/Four/Events/EventTypesResponse.cs:10]: Same as previous.

File: src/CheckoutSdk/Workflows/Four/Events/SubjectEvent.cs [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace Checkout.Workflows.Four.Events
+{
+    public class SubjectEvent : Resource
+    {
+        public string Id { get; set; }
+
+        public string Type { get; set; }
+
+        public string Timestamp { get; set; }
```
Comment (Armando-Ibarra) [inline src/CheckoutSdk/Workflows/Four/Events/SubjectEvent.cs:11]: Can't this be a DateTime?

File: src/CheckoutSdk/Workflows/Four/Rreflows/ReflowResponse.cs [hunk 6]
```diff
@@ -0,0 +1,14 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Checkout.Workflows.Four.Reflows
+{
+    public class ReflowResponse
+    {
+        [JsonProperty(PropertyName = "request_id")] public string RequestId { get; set; }
```
Comment (Armando-Ibarra) [inline src/CheckoutSdk/Workflows/Four/Rreflows/ReflowResponse.cs:8]: Same here, this serialization is already handled.

**Review by Armando-Ibarra — CHANGES_REQUESTED** [review]
Nice work.
Rebase your branch with master, there are a lot of changes that are not part of this task and makes difficult to review the PR please.

**Comment by AleGladchteinParser** [comment]
I close this PR because I need to create a new branch with changes for the workflows module.

**Pull request closed without merging**